source/git.js

```diff
@@ -48,6 +51,14 @@ GitExecutionTask.prototype.timeout = function(timeout) {
   this._timeout = timeout;
   return this;
 }
+GitExecutionTask.prototype.setStdoutListener = function(stdoutListener) {
```

Comment:
Looks like `setStdoutListener`,`setStderrListener`, `stdoutListener` and `stderrListener` are never used and can be removed.

(I think they were from your previous try with vi commands.)